stripe/proguard-rules.txt

```diff
@@ -2,4 +2,5 @@
 -keep class android.support.design.widget.CollapsingTextHelper { *; }
 -keep class org.bouncycastle.** { *; }
 
+-keep class com.stripe.android.** { *; }
```

Comment:
do we really need to keep every single class? this seems like a blanket rule :)

Reply:
Yeah, we should probably revisit this!